lib/jekyll/frontmatter_defaults.rb

```diff
@@ -130,7 +130,7 @@ def valid?(set)
     # new_scope - the new scope hash
     #
     # Returns true if the new scope has precedence over the older
-    def has_precedence?(old_scope, new_scope)
+    def precedence?(old_scope, new_scope)
```

Comment:
I'm gonna caution on the side of it not being entirely clear to the user,  things like Rubocop tell you to only use `private` once but that's an ignorant approach considering I didn't even know it was private until I scrolled up quite a ways.  They also tell you to indent on that private, which just muddies the code view.

I'm not against the change though, just saying it might not be clear to a user.